packages/perseus/src/components/lint.tsx

```diff
@@ -1,3 +1,47 @@
+import {StyleSheet, css} from "aphrodite";
+import * as React from "react";
+import ReactDOM from "react-dom";
+
+import * as constants from "../styles/constants";
+
+import InlineIcon from "./inline-icon";
+
+const exclamationIcon = {
+    path: "M6 11a1 1 0 1 1 0-2 1 1 0 0 1 0 2zm0-9a1 1 0 0 1 1 1v4a1 1 0 1 1-2 0V3a1 1 0 0 1 1-1z",
+    height: 12,
+    width: 12,
+} as const;
+
+enum Severity {
+    Error = 1,
+    Warning = 2,
+    Recommendation = 3,
+    OfflineReportingOnly = 4,
+}
```

Comment:
Can you check to see what the output of `flowgen` is for this?  I'm curious to see what it did with this.

Reply:
I checked and TS `enum` should work just fine with `flowgen`.  See #typescript-migration in Slack for deets.